components/low_power_mode/src/low_power_mode.c

```diff
@@ -0,0 +1,15 @@
+#include "low_power_mode.h"
+#include <esp_sleep.h>
+
+void low_power_mode_set_sleep_time(int sleep_time_sec)
+{
+    printf("Enabling timer wakeup, %ds\n", sleep_time_sec);
+    ESP_ERROR_CHECK(esp_sleep_enable_timer_wakeup(sleep_time_sec * 1000000));
+}
+
+void low_power_mode_enter_deep_sleep(void)
+{
+    printf("Entering low power mode\n");
```

Comment:
Please use ESP_LOGx instead of naked printf. See `esp_log.h` for details.

Example:
```
#include <esp_log.h>

const static char *TAG = "LOW_POWER_MODE_MANAGER";

void low_power_mode_set_sleep_time(int sleep_time_sec)
{
    ESP_LOGI(TAG, "Enabling timer wakeup, %ds\n", sleep_time_sec);
    ESP_ERROR_CHECK(esp_sleep_enable_timer_wakeup(sleep_time_sec * 1000000));
}
```